readme.md

```diff
@@ -1,62 +1,28 @@
-products:
-- windows
----
 
-# Windows Community Toolkit
+# Windows Community Toolkit :briefcase:
 The Windows Community Toolkit is a collection of helper functions, custom controls, and app services. It simplifies and demonstrates common developer patterns when building experiences for Windows 10.
 
-## Build Status
 | Target | Branch | Status | Recommended package version |
 | ------ | ------ | ------ | ------ |
 | Production | rel/6.1.0 | [![Build Status](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_apis/build/status/Toolkit-CI?branchName=rel/6.1.0)](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_build/latest?definitionId=10&branchName=rel/6.1.0) | [![NuGet](https://img.shields.io/nuget/v/Microsoft.Toolkit.Uwp.svg)](https://www.nuget.org/profiles/Microsoft.Toolkit) | 
 | Pre-release beta testing | master | [![Build Status](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_apis/build/status/Toolkit-CI?branchName=master)](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_build/latest?definitionId=10) | [![MyGet](https://img.shields.io/dotnet.myget/uwpcommunitytoolkit/vpre/Microsoft.Toolkit.Uwp.svg)](https://dotnet.myget.org/gallery/uwpcommunitytoolkit) |
 
-## Getting Started
-Please read the [getting Started with the Windows Community Toolkit](https://docs.microsoft.com/windows/communitytoolkit/getting-started) page for more detailed information about using the toolkit.
+## Getting Started :raised_hands:
+Please read the [Getting Started with the Windows Community Toolkit](https://docs.microsoft.com/windows/communitytoolkit/getting-started) page for more detailed information about using the toolkit.
 
-## Documentation
+## Documentation :pencil:
 All documentation for the toolkit is hosted on [Microsoft Docs](https://docs.microsoft.com/windows/communitytoolkit/). All API documentation can be found at the [.NET API Browser](https://docs.microsoft.com/dotnet/api/?view=win-comm-toolkit-dotnet-stable).
 
-## Windows Community Toolkit Sample App
+## Windows Community Toolkit Sample App :iphone:
 Want to see the toolkit in action before jumping into the code? Download and play with the [Windows Community Toolkit Sample App](https://www.microsoft.com/store/apps/9nblggh4tlcq) from the Store.
 
-## NuGet Packages
-NuGet is a standard package manager for .NET applications which is built into Visual Studio. To open the UI, from your open solution, choose the *Tools* menu > *NuGet Package Manager* > *Manage NuGet packages for solution...* . Enter one of the package names below to search for it online.
+## NuGet Packages :package:
+NuGet is a standard package manager for .NET applications which is built into Visual Studio. From your open solution, choose the *Tools* menu > *NuGet Package Manager* > *Manage NuGet packages for solution...* Enter one of the package names mentioned in [Windows Community Toolkit Nuget Packages](https://docs.microsoft.com/en-us/windows/communitytoolkit/nuget-packages) table to search for it online.
```

Comment:
It feels wordy and seems like it lacks context. We just tell them how to get the packages with out saying "To get our nuget packages...". 

Maybe instead of telling the user how to install NuGet our packages, we should let them know our NuGet package IDs. Or just link to the project wiki and list the IDs there. From what I understand how to get NuGet packages from nuget.org is a fairly common knolege in the community. Also how to obtain nuget packages [depends on your eviorment](https://docs.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow).

Reply:
The description is mirroring the [Nuget Package](https://docs.microsoft.com/en-us/windows/communitytoolkit/nuget-packages) page. I can work on the proper wording. 

At this point, they are just setting up and people come from all different backgrounds so I am not certain if we need this much detail here in the readme.  I am not certain about the ID's. Perhaps if you want to create a list in the wiki and then we can see if we can link that in the readme?

Reply:
O I am not saying to copy that page in the readme. Just maybe keep it shorter and not trying to explain to the users how to install the packages. Most users will probably just want the package IDs. So maybe making it more succinct and linking to the Nuget Package page. And maybe there we can make further improvements.

Reply:
Just fixed the wording for steps. It is already concise enough as this is all the information they need and the whole focus of this readme is to provide a glimpse of the information on steps and redirect it to the NuGet package. 

If you have a suggestion, create a commit suggestion and we can review if that's what we want in the content 😊

Reply:
Want to get the rest updated, @RosarioPulella feel free to submit a follow-on PR with any suggestions for changes.